packages/console/src/App.tsx

```diff
@@ -60,6 +60,20 @@ export const App = (): React.ReactElement => {
       });
     }
   });
+  const routes: IRoute[] = [
+    { path: '/canvas', page: CanvasPage },
+    { path: '/canvas-preview', page: CanvasPreviewPage },
+    { path: '/', page: HomePage },
+    { path: '/accounts/:accountId', page: AccountPage, redirectIfNoAuth: '/login' },
```

Comment:
you need to do the same props thing from web3images on all these pages that require props